resources/keywords/import_provider_account.resource

```diff
@@ -257,7 +259,7 @@ Import ${databaseProvider} Provider Account
     ...    User Enters Data To Import CrunchyDB Provider Account
     ...  ELSE IF    "Cockroach" in """${databaseProvider}"""
     ...    User Enters Data To Import CockroachDB Provider Account
-    ...  ELSE IF    "RDS" in """${databaseProvider}"""
+    ...  ELSE IF    "RDS" in """${DBProvider}"""
```

Comment:
@ikanias Can we keep it as ${databaseProvider} to have uniformity?

Reply:
Sure!